distributed/distributed.yaml

```
@@ -74,6 +74,13 @@ distributed:
   deploy:
     lost-worker-timeout: 15s  # Interval after which to hard-close a lost worker job
 
+  adaptive:
+    interval: 1s # interval between scaling evaluations
+    target-duration: 5s # time an entire graph calculation is desired to take ("1m", "30m")
+    minimum: 0 # minimum number of workers
+    maximum: 100000 # maximum number of workers
+    wait-count: 3 # number of times a worker should be suggested for removal before removing it
```

Style nitpicks:

```suggestion
    interval: 1s         # Interval between scaling evaluations
    target-duration: 5s  # Time an entire graph calculation is desired to take ("1m", "30m")
    minimum: 0           # Minimum number of workers
    maximum: 100000      # Maximum number of workers
    wait-count: 3        # Number of times a worker should be suggested for removal before removing it
```

Can the maximum workers be changed to `inf`?

---

Thanks for the change in the config. Yes, i will change that back to inf again.